TenureInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -16,14 +17,14 @@ public static TenureResponseObject ToResponse(this TenureInformation domain)
                 Charges = domain.Charges,
                 EndOfTenureDate = domain.EndOfTenureDate,
                 EvictionDate = domain.EvictionDate,
-                HouseholdMembers = domain.HouseholdMembers.ToList(),
+                HouseholdMembers = domain.HouseholdMembers.ToListOrEmpty(),
```

Comment:
does it need to be an empty list, or could you use `domain.HouseholdMembers?.ToList()`

Reply:
We could do either, and with no instruction either way I went with what other bits of the code was already doing so as to ensure as little impact on the overall functionality as possible.